src/args/arg_builder/flag.rs

```diff
@@ -67,7 +67,7 @@ impl<'n, 'e> AnyArg<'n, 'e> for FlagBuilder<'n, 'e> {
         self.b.requires.as_ref().map(|o| &o[..])
     }
     fn blacklist(&self) -> Option<&[&'e str]> { self.b.blacklist.as_ref().map(|o| &o[..]) }
-    fn required_unless(&self) -> Option<&[&'e str]> { None }
+    fn required_unless(&self) -> Option<&[&'e str]> { self.b.r_unless.as_ref().map(|o| &o[..]) }
```

Comment:
This line is copied from the analogous line in the Positional arg builder: https://github.com/kbknapp/clap-rs/blob/f85fd9fbc8a1b2f5f319dad272405ac8e43c790f/src/args/arg_builder/positional.rs#L141 - I assume that the logic is supposed to be the same for both types of args.